extension-manifest-v3/src/utils/api.js

```diff
@@ -100,19 +148,19 @@ export async function session() {
     }
   }
 
-  return jwtDecode(accessToken);
+  return jwtDecode(accessToken.value);
 }
 
 export async function getUserOptions() {
   const userId = await getCookie('user_id');
   const accessToken = await getCookie('access_token');
   const csrfToken = await getCookie('csrf_token');
 
-  const res = await fetch(`${ACCOUNT_URL}/options/${userId}`, {
+  const res = await fetch(`${ACCOUNT_URL}/options/${userId.value}`, {
```

Comment:
in case the userId cookie is falsy this will result in not very readable error. Shouldn't you check if the cookie exist before you use it value?

Reply:
Yes, and we do. To avoid multiple calls to `session()` it is done out side of this function. This one assumes that user is logged in correctly.